pkg/ccl/streamingccl/streamingest/replication_stream_e2e_test.go

```diff
@@ -1143,31 +1143,25 @@ func TestTenantStreamingRetryLoadJob(t *testing.T) {
 	c, cleanup := replicationtestutils.CreateTenantStreamingClusters(ctx, t, args)
 	defer cleanup()
 
+	// Inject an error to fail the resumer.
+	mu.Lock()
+	knobLoadErr = errors.Newf("test error")
```

Comment:
any reason to move this before WaitUntilReplicatedTime so its an actual injection and not an immediate failure?

Reply:
I wanted to cause a retry on the first producer job resume call.